integrationTest/integration_test.go

```diff
@@ -1612,6 +1618,12 @@ func makeTests() []*TestGroup {
 			tc("Create DNAME record in non-FQDN", dname("a", "b")),
 		),
 
+		testgroup("DNSKEY",
+			requires(providers.CanUseDNSKEY),
+			tc("Create DNSKEY record", dnskey("test", 257, 3, 13, "AABBCCDD")),
+			tc("Modify DNSKEY record", dnskey("test", 256, 3, 13, "AABBCCDDEEFF")),
```

Comment:
Please modify each field one at a time (if possible).

Are those values valid for DNSKEY?  (I haven't checked the RFC... they might be valid).  If not, please only use valid examples. Some DNS providers validate the fields and will reject our tests.